includes/class-wc-payments-payment-request.php

```diff
@@ -88,6 +89,22 @@ public function __construct( WC_Payments_Account $account ) {
 		$this->init();
 	}
 
+	/**
+	 * Payment request default settings.
+	 *
+	 * @return array Default settings.
+	 */
+	public static function get_default_settings() {
+		return [
+			'payment_request'                     => 'yes',
```

Comment:
Would it be simple enough to get these directly from the gateway's form field definitions? ([Here](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/aa7569b6fd8138b8d75cc2e8bbe6971200dd6d3a/includes/connect/class-wc-stripe-connect.php#L169-L183)'s an example of that from the Stripe extension.)

Reply:
I thought about syncing the other way around but didn't find any examples when looking at the settings default values.
Thanks for pointing out the example.

I'll try to do the same thing as in Stripe.

Reply:
Note that that code is pretty new to the Stripe extension (migrated from WooCommerce Services pretty recently), and it's not used for this kind of purpose (though perhaps it should be, not sure).

Reply:
I just noticed we'll need to instantiate all these + WC_Payment_Gateway_WCPay just to get the form_fields.

```php
public function __construct(
    WC_Payments_API_Client $payments_api_client,
    WC_Payments_Account $account,
    WC_Payments_Customer_Service $customer_service,
    WC_Payments_Token_Service $token_service,
    WC_Payments_Action_Scheduler_Service $action_scheduler_service
)
```

What if we do the other way around? Something like this would look cleaner:

```php
$payment_request_default = WC_Payments_Payment_Request::get_default_settings();
```

And then in form_fields

```php
'payment_request_button_type' => [
    // ...
    'default' => $payment_request_default['payment_request_button_type'],
```

Reply:
Ah, yeah, that looks good to me 👍